app/models/post_custom_prompt.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class PostCustomPrompt < ActiveRecord::Base
```

Comment:
I think this is really similar to the existing `CompletionPrompt`model. What do you think about changing that instead of creating a second one?

Reply:
sure will have a look!

Reply:
I think we should possibly attack this later, it just is much leaner than CompletionPrompt... if we were to add this there it can get messy - we are going to want a polymorphic association and so on... I think the post specific one is a bit easier to reason about.